vars/fhcapProviderConfigFor.groovy

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/groovy
+
+def call(providerId, providers, template = 'default') {
+    def providerCfg = null
+    String delimiter = '-'
+    List<String> tmplParts = template.split(delimiter) as List<String>
+    while(tmplParts.size() > 0) {
```

Comment:
If you're going to make any changes to this before merging, could you also put a space after the `while` keyword? (same with the `if` on line 10)